build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/plugin_list.rb

```diff
@@ -40,6 +40,7 @@ def compute_plugins(options)
     def default_plugins
       {
         'Seahorse::Client::Plugins::ContentLength' => "#{seahorse_plugins}/content_length.rb",
+        'Seahorse::Client::Plugins::ContentType' => "#{seahorse_plugins}/content_type.rb",
```

Comment:
With this being default, should we relocate the setting of `content-type` for other protocols like [json](https://github.com/aws/aws-sdk-ruby/blob/09ee4c3e0b96e807d9fd1b9331c4387746f0896f/gems/aws-sdk-core/lib/aws-sdk-core/json/handler.rb#L23) here?